website/docs/docs/cloud-integrations/downstream-exposures-tableau.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Configure exposures automatically"
+sidebar_label: "Configure exposures automatically"
+description: "Configure and visualize exposures automatically to auto-generate them from Tableau dashboards, helping you understand how models are used in downstream tools for a richer downstream lineage"
+image: /img/docs/cloud-integrations/auto-exposures/explorer-lineage2.jpg
+---
+
+# Configure exposures automatically <Lifecycle status="enterprise" />
+
+<IntroText>
+Configure and automatically populate downstream exposures for supported BI tool integrations. Visualize and orchestrate them through <a href="https://docs.getdbt.com/docs/collaborate/explore-projects">dbt Explorer</a> and the [dbt Cloud job scheduler](/docs/deploy/job-scheduler) for a richer experience.
+
+</IntroText>
+
+As a data team, it’s critical that you have context into the downstream use cases and users of your data products. In dbt Cloud, you can configure downstream exposures in two ways:
+- Manual &mdash; Defined [manually](/docs/build/exposures#declaring-an-exposure) and explicitly in your project’s YAML files.
+- Automatic &mdash; Pulled automatically for supported dbt Cloud integrations. dbt Cloud automatically creates and visualizes downstream exposures, removing the need for manual YAML definitions. These downstream exposures are stored in dbt’s metadata system, appear in [dbt Explorer](/docs/collaborate/explore-projects), and behave like manual exposures, however they don’t exist in YAML files.
+
+By leveraging downstream [exposures](/docs/build/exposures) automatically, data teams can:
+
+- Gain a better understanding of how models are used in downstream analytics, improving governance and decision-making.
```

Comment:
This section might be better served on line 15 after "As a data team, it’s critical that you have context into the downstream use cases and users of your data products."

That way we're setting the pins up and knocking them down before we dive into the setup and fleshing out our lead in